geoh5py/objects/surface.py

```diff
@@ -34,45 +34,53 @@ class Surface(CellObject):
         fields=(0xF26FEBA3, 0xADED, 0x494B, 0xB9, 0xE9, 0xB2BBCBE298E1)
     )
 
-    def __init__(self, object_type: ObjectType, **kwargs):
-        self._cells: np.ndarray | None = None
+    @classmethod
+    def default_type_uid(cls) -> uuid.UUID:
+        return cls.__TYPE_UID
 
-        super().__init__(object_type, **kwargs)
+    def validate_cells(self, indices: list | tuple | np.ndarray | None):
```

Comment:
Add docstring to this function

General comments: Do we want to keep the validators as public function, or pass them as private?

Reply:
I think it could be useful down the road to ask the object to validate an array ahead of time. Not sure making it private does anything more.,